app/google-services.json

```diff
@@ -39,12 +47,12 @@
       "client_info": {
         "mobilesdk_app_id": "1:213185384651:android:7d519dc202f8de90f3cb4e",
         "android_client_info": {
-          "package_name": "com.github.campus_capture.sdpfirebase"
+          "package_name": "com.github.jenjamin3000.sdpfirebase"
```

Comment:
This is linked to the Firebase service and has no connection with any of our app's local files, right? Otherwise this package name would have been refactored since then.